Source/WebCore/rendering/RenderVideo.cpp

```diff
@@ -294,6 +295,15 @@ void RenderVideo::updateFromElement()
     updatePlayer();
 }
 
+bool RenderVideo::inFullscreenElementOrFullscreenVideo() const
```

Comment:
Nit: "VideoFullscreen" is used more commonly used than "FullscreenVideo", so maybe `inFullscreenElementOrVideoFullscreen`?

Reply:
Sounds good, I'll change that.